tests/benchmark/runner/src/benchmarkCli.ts

```diff
@@ -0,0 +1,39 @@
+import { config } from './common';
+import * as yargs from 'yargs';
+import * as fs from 'fs';
+import { runBench } from './benchmarkRunner';
+
+let args = yargs(process.argv)
+	.usage(
+		'$0 [--framework Framework1,Framework2,...] [--benchmark Benchmark1,Benchmark2,...] [--count n] [--exitOnError]'
+	)
+	.help('help')
+	.default('check', 'false')
+	.default('exitOnError', 'false')
+	.default('count', config.REPEAT_RUN)
+	.boolean('headless')
+	.array('framework')
+	.array('benchmark').argv;
+
+console.log(args);
```

Comment:
do we need this console.log here?

Reply:
Nope! Sorry I just caught that but hadn't pushed it